src/index.ts

```diff
@@ -109,13 +121,21 @@ export function viteSvgToWebfont<T extends GeneratedFontTypes = GeneratedFontTyp
             }
             await generate();
             if (isBuild && !options.inline) {
-                const emitted = processedOptions.types.map<[T, string]>(type => [
-                    type,
-                    `/${this.getFileName(this.emitFile({ type: 'asset', fileName: `assets/${processedOptions.fontName}-${guid()}.${type}`, source: generatedFonts?.[type] }))}`,
-                ]);
+                const emitted = processedOptions.types.map<[T, string]>(type => {
+                    if (!generatedFonts?.[type]) {
+                        throw new Error(`Failed to generate font of type ${type}`);
+                    }
+
+                    const fileContents = Buffer.from(generatedFonts[type]);
+                    const hash = getBufferHash(fileContents);
+                    const filePath = pathJoin(TMP_DIR, `${processedOptions.fontName}-${hash}.${type}`);
```

Comment:
I've replaced the `guid()` call with hash based on a file contents. Technically speaking, there's no need to add some uniqueness here because the tmp dir is already unique, but it will make easier name-based comparation later in `generateBundle()` hook (which is required to keep external API working).